### PR TITLE
Python_MySQL assgn

### PR DESCRIPTION
**Pull request by Priya-661**

### CODE REVIEW

File: Python_MySQL/Delete-Record.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+user = "root", 
+password = "MySQL@$123", 
+database = "myassignment"
+) 
+
+
+mycursor = myDB.cursor() 
+
+
+query = "DELETE from Courses WHERE Course_id = 3"
```
Comment (sgpratik) [inline Python_MySQL/Delete-Record.py:14]: Also, read about parameterising the queries.

File: Python_MySQL/Create-Table.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  user="root",
+  password="MySQL@$123",
+  database="myassignment"
+)
+
+mycursor = myDB.cursor()
+
+CoursesRecord= """CREATE TABLE Courses (
+                   Course_id  INT ,
+                   Course_name VARCHAR(50),
```
Comment (sgpratik) [inline Python_MySQL/Create-Table.py:14]: For column name conventions follow the snake case.

File: Python_MySQL/Count.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+user = "root", 
+password = "MySQL@$123", 
+database = "myassignment"
+) 
+
+
+mycursor = myDB.cursor() 
+
+
+query = "Select COUNT(*) As Total_Courses From Courses"
```
Comment (sgpratik) [inline Python_MySQL/Count.py:14]: for SQL commands either keep it all upper or lower case. Don't mix.

File: Python_MySQL/Add-column.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import mysql.connector 
+
+myDB = mysql.connector.connect( 
+host = "localhost", 
+user = "root", 
+password = "MySQL@$123", 
+database = "myassignment"
```
Comment (sgpratik) [inline Python_MySQL/Add-column.py:7]: read about dotenv here, https://pypi.org/project/python-dotenv/

Never push the credentials or any sensitive information to the repository.

File: Python_MySQL/Count.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import mysql.connector 
+
+myDB = mysql.connector.connect( 
+host = "localhost", 
+user = "root",
```
Comment (sgpratik) [inline Python_MySQL/Count.py:5]: Don't create a session every time. Create it in one place and re-use it.

**Review by sgpratik — CHANGES_REQUESTED** [review]
Work on the suggestions on the PR.

File: Python_MySQL/Count.py (hunk 5 above)
Comment (sgpratik) [reply Python_MySQL/Count.py:5]: You have to create a connection and close for every transaction. You'll also be able to read about using context managers for the connection.

File: Python_MySQL/Add-column.py (hunk 4 above)
Comment (Priya-661) [reply Python_MySQL/Add-column.py:7]: Yeah, Sure

File: Python_MySQL/Count.py (hunk 5 above)
Comment (Priya-661) [reply Python_MySQL/Count.py:5]: Ok

File: Python_MySQL/Count.py (hunk 3 above)
Comment (Priya-661) [reply Python_MySQL/Count.py:14]: OK

File: Python_MySQL/Delete-Record.py (hunk 1 above)
Comment (Priya-661) [reply Python_MySQL/Delete-Record.py:14]: Sure.

File: Python_MySQL/Create-Table.py (hunk 2 above)
Comment (Priya-661) [reply Python_MySQL/Create-Table.py:14]: Ok

**Pull request closed without merging**